src/plugins/vis_type_table/public/table_vis_renderer.tsx

```diff
@@ -3,14 +3,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from 'react';
+import React, { lazy } from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 
+import { CoreStart } from 'opensearch-dashboards/public';
 import { VisualizationContainer } from '../../visualizations/public';
 import { ExpressionRenderDefinition } from '../../expressions/common/expression_renderers';
 import { TableVisRenderValue } from './table_vis_fn';
+// @ts-ignore
```

Comment:
Why do we need to ignore this and why are we lazy loading this?

Reply:
I will remove the `//@ts-ignore`. The previous type complain is gone.

```
Could not find a declaration file for module './vis_editor'. '/home/ubuntu/OpenSearch-Dashboards/src/plugins/vis_type_table/public/components/table_vis_app' implicitly has an 'any' type.ts
```

For the lazy loading one, this is an interesting. If remove the lazy load, we will get type complain:

```
'TableVisApp' cannot be used as a JSX component.
  Its return type 'Promise<typeof import("/home/ubuntu/OpenSearch-Dashboards/src/plugins/vis_type_table/public/components/table_vis_app")>' is not a valid JSX element.
    Type 'Promise<typeof import("/home/ubuntu/OpenSearch-Dashboards/src/plugins/vis_type_table/public/components/table_vis_app")>' is missing the following properties from type 'ReactElement<any, any>': type, props, keyts(2786)
```

I see multiple places in our code base for this complain if remove lazy load. Seems to me, we use lazy load not for performance purpose but compile purpose.